### ISSUE TITLE
parseSailsConfig() does not pass multipleStatements to DATABASE_URL

### ISSUE THREAD

**Issue by junamai2000** [building5/sails-db-migrate#28]
Hi,

I want to use SQL for migration.
My migration script contains multiple queries, but sails-db-migrate does not path multipleStatements to DATABASE_URL.

Can you fix the issue or is there anyway to execute multiple queries?

In my connections.js, I have a following setting.
```
multipleStatements: true
```
but in the init process, it ignores the setting above.

I think lib/sailsDbMigrate.js does not take the setting. 

Thanks.

**Comment by leedm777** [comment]
I would accept a PR that adds that feature.

If you'd like to just handle it yourself in the migrations, you can just call `runSql()` multiple times:

```javascript
db.runSql('CREATE TABLE foo...', function (err, res) => {
  if (err) { return cb(err); }
  db.runSql('CREATE TABLE bar...', function (err, res) => {
    if (err) { return cb(err); }
    ...
```

Or you you could use [`async.series()`](https://github.com/caolan/async#seriestasks-callback) to avoid nested callbacks.

```
async.series([
  db.bind(db, 'CREATE TABLE foo...'),
  db.bind(db, 'CREATE TABLE bar...'),
}, cb);
```

**Comment by junamai2000** [comment]
Actually, I have a huge sql to run. your solution does not work in my case. So, I made PR. Could you check it?

**Comment by leedm777** [comment]
Merged in v1.4.0. Thanks!
